plate_simulation/driver.py

```diff
@@ -239,17 +240,38 @@ def make_model(self) -> FloatData:
             history=[dikes, overburden, erosion],
         )
 
-        geology = scenario.geologize()
-
-        if self.simulation_parameters.physical_property == "conductivity":
```

Comment:
Is this in scenario geologize now? 
Or this option was removed?

Reply:
I refactored out the Scenario class and just combined with Geology. Not sure we needed this middle man since it complicated the types on the base `Series` class.
See what you think.